src/Button/Button.tsx

```diff
@@ -91,11 +94,10 @@ const Button = styled('button')<ButtonProps>`
 `;
 
 Button.defaultProps = {
+  color: 'gray',
   fullWidth: false,
-  isDisabled: false,
```

Comment:
This prop was unused, so unless we're building this one out into a full blown component (so it can do things like `isLoading` and `iconStart` or `iconEnd`), then the `disabled` prop works fine